docs/hardhat.md

```diff
@@ -0,0 +1,65 @@
+Hardhat Integration
+============
+
+_(Supported since Remix IDE v0.12.0 and Remixd v0.3.6)_
+
+Remixd and Hardhat
+------------------
+
+**Note:** If you have not used `remixd` before, read more about it [here](./remixd.html)
+
+If `remixd` is running locally on device and shared folder is a **hardhat project**, one more websocket listener will be running on port `65522`. According to its documentation,
```

Comment:
If `remixd` is running locally on your device and the shared folder is a hardhat project, an additional websocket listener will be running on port 65522.

Reply:
updated